content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,139 @@
 
-To delete a variable, click its "🗑" (trash can) icon, then confirm your decision in the dialog box that appears.
+~> **Note:** Terraform Cloud can infer default variable values, but it does not automatically display those variable names in the workspace UI.
 
-You can edit one variable at a time, and must save or cancel your current edits before editing other variables in the list.
+## Scope
 
-![Screenshot: A variable being edited](./images/vars-edit.png)
+Each environment and Terraform variable can have one of the following scopes:
 
-### Multi-line Values
+| Scope | Description| Resources |
+|-------|------------|-----------|
+|Workspace-Specific | Apply to a single workspace | [Create Workspace-Specific Variables](/docs/cloud/workspaces/managing-variables.html#workspace-specific-variables), [Loading Variables from Files](#loading-variables-from-files), [Workspace-Specific Variables API](/docs/cloud/api/workspace-variables.html).|
+|Variable Set | Apply to multiple workspaces within the same organization. | [Create Variable Sets](/docs/cloud/workspaces/managing-variables.html#variable-sets) and [Variable Sets API](/docs/cloud/api/variable-sets.html)|
+|Global Variable Set | Automatically applied to all current and future workspaces within an organization. | [Create Variable Sets](/docs/cloud/workspaces/managing-variables.html#variable-sets) and [Variable Sets API](/docs/cloud/api/variable-sets.html)|
 
-The text fields for variable values can handle multi-line text (typed or pasted) without any special effort.
 
-### HCL Values
 
-Variable values are strings by default. To enter list or map values, click the variable's "HCL" checkbox (visible when editing) and enter the value with the same HCL syntax you would use when writing Terraform code. For example:
+## Precedence
 
-```hcl
-{
-    us-east-1 = "image-1234"
-    us-west-2 = "image-4567"
-}
-```
+> **Hands On:** The [Manage Multiple Variable Sets in Terraform Cloud](https://learn.hashicorp.com/tutorials/terraform/manage-variable-sets) tutorial on HashiCorp Learn shows how to manage multiple variable sets and demonstrates variable precedence.
 
-HCL can be used for Terraform variables, but not for environment variables. The HCL code you enter for values is interpreted by the same Terraform version that performs runs in the workspace. (See [How Terraform Cloud Uses Variables](#how-terraform-cloud-uses-variables) below.)
 
-### Sensitive Values
+There may be cases when a workspace contains conflicting variables of the same type with the same key. Terraform Cloud marks overwritten variables in the UI.
 
-Terraform often needs cloud provider credentials and other sensitive information that shouldn't be widely available within your organization.
+![The Terraform Cloud UI indicates which variables are overwritten](/docs/cloud/workspaces/images/ui-overwritten-variables.png)
```

Comment:
Is this the right resolution?

Reply:
it actually was, but it looked weird I think because of the way I tried to zoom into a tiny part of the screen - I redid this one and it looks much better now!